sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc

```diff
@@ -38,6 +38,18 @@ Previously, {sandboxed-containers-first} availability on AWS bare metal was in T
 [id="sandboxed-containers-1-3-bug-fixes"]
 == Bug fixes
 
+* Previously, when installing {sandboxed-containers-first} on {product-title} 4.10, the controller manager POD failed to start with the following error:
++
+[source,terminal]
+----
+container has runAsNonRoot and image has non-numeric user (nonroot),
+          cannot verify user is non-root
+----
++
+Because of this issue, the `KataConfig` CR could not be created and {sandboxed-containers-first} could not run.
```

Comment:
```suggestion
Because of this issue, the `KataConfig` CR could not be created and the {sandboxed-containers-first} could not run.
```

Reply:
@skrthomas - no article needed here. The variable is for the product: Because of this issue, the `KataConfig` CR could not be created and OpenShift sandboxed containers could not run.